src/components/BookCard.tsx

```diff
@@ -0,0 +1,17 @@
+import { IBook } from '../interfaces/books';
+
+const BookCard = (key: any, book: IBook) => {
+
+    return (
+        <div key={key} className="book">
+            <img src="../images/book.jpg"
+                alt={book.title}
```

Comment:
Nit - if the image does not have a meaningful alt and it is just repeating something that already exists in elsewhere, the blind user will hear it twice/more. You can maybe use `${book.title} cover}`, though I am not sure how helpful that would be to the blind user. To mark an image as presentational (and so screen readers would ignore it), you can use an empty string alt - `alt=""`